aten/src/ATen/native/native_functions.yaml

```diff
@@ -1672,7 +1672,7 @@
   dispatch:
     CompositeExplicitAutograd: vdot_out
 
-- func: einsum(str equation, Tensor[] tensors) -> Tensor
+- func: einsum(str equation, Tensor[] tensors, *, int[]? optimize=None) -> Tensor
```

Comment:
What's the current FC support for adding a new kwarg with a default value? Will old scripted programs that call einsum work OK? If someone exports the new einsum to an older version of PyTorch, will that work?